### DISCUSSION TITLE
Motion to remove tests from this project

### DISCUSSION THREAD

**Discussion by davidsaulrodriguez** [davidsaulrodriguez/shop-portal#29] [General]
I want to make a motion to remove the test from this project.

TDD (test driven development) will take entirely too long to get done in such a short time relative to this project's due date.

For now I vote to remove all test files from the project.

This is completely up for discussion

@jdavidrice @lucpizz @Mconway409 @pennyquirino @Delph-Sunny

**Comment by jdavidrice** [comment]
I’m fine with removing tests. I didn’t think having them in the repo would be a big deal, but if removing them completely moves things along, then by all means let’s forget them.

Jeremy
